.github/workflows/quick-start.yml

```diff
@@ -13,4 +13,23 @@ jobs:
   open-api-workflow-quick-start:
     uses: maykinmedia/open-api-workflows/.github/workflows/quick-start.yml@v5
     with:
-      fixtures: 'klantinteracties contactgegevens'
\ No newline at end of file
+      fixtures: 'klantinteracties contactgegevens'
+
+  upgrade_simulation:
+    name: Simulate upgrading instances
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_OLD: "2.5.1"
+      RELEASE_NEW: "2.8.0"
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build and start docker with RELEASE=${{ env.RELEASE_OLD }}
+        run: |
+          docker compose build --build-arg RELEASE=${{ env.RELEASE_OLD }} || ( docker compose logs >&2 && exit 1 )
+          docker compose up -d || ( docker compose logs >&2 && exit 1 )
```

Comment:
you could also redirect the logs to the step summary, see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#adding-a-job-summary which make them easier accessible to debug